assets/terraform-modules/matchbox-flatcar/variables.tf

```
@@ -0,0 +1,74 @@
+# bare-metal
+variable "http_endpoint" {
+  type        = string
+  description = "Matchbox HTTP read-only endpoint (e.g. http://matchbox.example.com:8080)"
+}
+
+variable "os_channel" {
+  type        = string
+  default     = "stable"
+  description = "Flatcar Container Linux channel to install from (stable, beta, alpha, edge)"
+}
+
+variable "os_version" {
+  type        = string
+  default     = "current"
+  description = "Flatcar Container Linux version to install (for example '2191.5.0' - see https://www.flatcar-linux.org/releases/)"
+}
+
+variable "download_protocol" {
+  type        = string
+  default     = "https"
+  description = "Protocol iPXE should use to download the kernel and initrd. Defaults to https, which requires iPXE compiled with crypto support. Unused if cached_install is true."
+}
+
+variable "cached_install" {
+  type        = bool
+  default     = false
+  description = "Whether the operating system should PXE boot and install from matchbox /assets cache. Note that the admin must have downloaded the os_version into matchbox assets."
+}
+
+variable "install_disk" {
+  type        = string
+  default     = "/dev/sda"
+  description = "Disk device to which the install profiles should install the operating system (e.g. /dev/sda)"
+}
+
+variable "container_linux_oem" {
+  type        = string
+  default     = ""
+  description = "DEPRECATED: Specify an OEM image id to use as base for the installation (e.g. ami, vmware_raw, xen) or leave blank for the default image"
+}
+
+variable "kernel_args" {
+  description = "Additional kernel arguments to provide at PXE boot."
+  type        = list(string)
+  default     = []
+}
+
+variable "install_to_smallest_disk" {
+  description = "Install Flatcar Container Linux to the smallest disk."
+  type        = bool
+  default     = false
+}
+
+variable "ssh_keys" {
+  type        = list(string)
+  description = "SSH public keys for user 'core'"
```

Would be nice to add missing dots.
```suggestion
  description = "SSH public keys for user 'core'."
```